.gitignore

```diff
@@ -1 +1,2 @@
-*.DS_Store
```

Comment:
> I agree, I like staying in project root as well. So then why this: https://github.com/gonebusy/gonebusy-php-client/pull/7/files#diff-04c6e90faac2675aa89e2176d2eec7d8R65

Ah good find. It's confusing because those are the simplified instructions to test aka try-out the SDK and they use the folder `test` but not related to unit tests, in the `tests` folder (:

Should we change it anyway to run it like `php test/testSDK.php` ? -- It really depends on the code users put in there.